Sprint-3/implement/get-angle-type.js

```diff
@@ -25,3 +25,50 @@
 // Identify Reflex Angles:
 // When the angle is greater than 180 degrees and less than 360 degrees,
 // Then the function should return "Reflex angle"
+
+function getAngleType(angle) {
+    // Check if the angle is exactly 90 degrees
+    if (angle === 90) {
+        return "Right angle";
+    }
+    // Check if the angle is less than 90 degrees
+    else if (angle < 90) {
+        return "Acute angle";
+    }
+    // Check if the angle is greater than 90 degrees and less than 180 degrees
+    else if (angle > 90 && angle < 180) {
+        return "Obtuse angle";
+    }
+    else if (angle === 180) {
+        return "Straight angle";
+    }
+    // Check if the angle is greater than 180 degrees and less than 360 degrees
+    else if (angle > 180 && angle < 360) {
+        return "Reflex angle";
+    }
+    //check if the angle is not an angle returns an error
+    else {
+        return "Invalid angle";
+    }
+}
+// Test the function with different angles
+console.log(getAngleType(90)); // Should print "Right angle"
+console.log(getAngleType(45)); // Should print "Acute angle"
+console.log(getAngleType(135)); // Should print "Obtuse angle"
+console.log(getAngleType(180)); // Should print "Straight angle"
+console.log(getAngleType(225)); // Should print "Reflex angle"
+console.log(getAngleType(360)); // Should print "Reflex angle"
+console.log(getAngleType(0)); // Should print "Invalid angle"
+console.log(getAngleType(-45)); // Should print "Invalid angle"
```

Comment:
Does these two show "Invalid angle"?

It is not easy to spot the error from just `console.log()` output. That's one reason why a test framework like Jest can help us identify cases when a return value does not match an expected value.

Reply:
I changed the testing method to Jest